vehicle/models/vehicle.py

```diff
@@ -0,0 +1,22 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models, _, api
+
+
+class Vehicle(models.Model):
+    _name = "vehicle.vehicle"
+    _description = "Vehicle"
+
+    name = fields.Char(default="/", readonly=True)
+    license_plate = fields.Char(required=True)
+    active = fields.Boolean(string="Active", default=True)
+    user_id = fields.Many2one(
+        comodel_name="res.users",
+        string="Responsible user",
+        default=lambda self: self.env.user,
+        required=True)
+
+    @api.constrains('license_plate')
+    def _constrains_name(self):
+        # Example 
+        self.name = "VVV-" + self.license_plate
```

Comment:
In this first version, it should be enough IMO:
```suggestion
        # Example 
        self.name = self.license_plate
```
We should improve this later